### PR TITLE
Docker job update w.r.t PS 5.7

### PR DESCRIPTION
**Pull request by kaushikpuneet07**
Skip component installation in PS 5.7

### CODE REVIEW

**Comment by kaushikpuneet07** [comment]
Successfull Job Run : https://ps57.cd.percona.com/view/QA/job/test-ps-docker-image/46/

File: docker-image-tests/ps/settings.py [hunk 1]
```diff
@@ -95,6 +95,7 @@
     ps_binaries = ps57_binaries
     ps_plugins = ps57_plugins
     ps_functions = ps57_functions
+    ps_components = ps80_components
```
Comment (eleo007) [inline docker-image-tests/ps/settings.py:98]: Why do we add "ps80_components" to ps 5.7 part?

File: docker-image-tests/ps/Jenkinsfile [hunk 2]
```diff
@@ -24,7 +24,7 @@ pipeline {
             export PATH=${PATH}:~/.local/bin
             sudo yum install -y python3 python3-pip
             rm -rf package-testing
-            git clone https://github.com/Percona-QA/package-testing.git --depth 1
+            git clone -b docker-job  https://github.com/kaushikpuneet07/package-testing.git --depth 1
```
Comment (eleo007) [inline docker-image-tests/ps/Jenkinsfile:27]: Change to default git instaed of local

File: docker-image-tests/ps/settings.py (hunk 1 above)
Comment (kaushikpuneet07) [reply docker-image-tests/ps/settings.py:98]: IF we don't define than it throws error :

```
==================================== ERRORS ====================================
__________________ ERROR collecting tests/test_ps_dynamic.py ___________________
tests/test_ps_dynamic.py:24: in <module>
    class TestDynamic:
tests/test_ps_dynamic.py:56: in TestDynamic
    @pytest.mark.parametrize("cmpt", ps_components)
E   NameError: name 'ps_components' is not defined
```

File: docker-image-tests/ps/tests/test_ps_dynamic.py [hunk 3]
```diff
@@ -53,10 +53,15 @@ def test_install_plugin(self, host, pname, soname):
         cmd = host.run('mysql --user=root --password='+ps_pwd+' -S/var/lib/mysql/mysql.sock -s -N -e "SELECT plugin_status FROM information_schema.plugins WHERE plugin_name = \''+pname+'\';"')
         assert cmd.succeeded
         assert 'ACTIVE' in cmd.stdout
+
+if ps_version_major in ['8.0']:
+#    pytestmark = pytest.mark.skipif(ps_version_major=['5.7'] ,reason="Components are supported from 8.0 onwards")
```
Comment (eleo007) [inline docker-image-tests/ps/tests/test_ps_dynamic.py:58]: Do we need this in final PR?

Comment (kaushikpuneet07) [reply docker-image-tests/ps/tests/test_ps_dynamic.py:58]: NO , will delete it

File: docker-image-tests/ps/tests/test_ps_dynamic.py [hunk 4]
```diff
@@ -53,10 +53,15 @@ def test_install_plugin(self, host, pname, soname):
...
+if ps_version_major in ['8.0']:
+#    pytestmark = pytest.mark.skipif(ps_version_major=['5.7'] ,reason="Components are supported from 8.0 onwards") 
     @pytest.mark.parametrize("cmpt", ps_components)
     def test_install_component(self, host, cmpt):
         cmd = host.run('mysql --user=root --password='+ps_pwd+' -S/var/lib/mysql/mysql.sock -s -N -e "INSTALL component \''+cmpt+'\';"')
         assert cmd.succeeded
         cmd = host.run('mysql --user=root --password='+ps_pwd+' -S/var/lib/mysql/mysql.sock -s -N -e "SELECT component_urn from mysql.component WHERE component_urn = \''+cmpt+'\';"')
         assert cmd.succeeded
         assert cmpt in cmd.stdout
+else:
```
Comment (eleo007) [inline docker-image-tests/ps/tests/test_ps_dynamic.py:66]: Do we actually need else part?

Comment (kaushikpuneet07) [reply docker-image-tests/ps/tests/test_ps_dynamic.py:66]: yes

**Comment by kaushikpuneet07** [comment]
successful jobs after update 
PS 57 : https://ps57.cd.percona.com/view/QA/job/test-ps-docker-image/55/
PS 80 : https://ps80.cd.percona.com/view/QA/job/test-ps-docker-image/180/

**Comment by kaushikpuneet07** [comment]
successful jobs after update
PS 57 : https://ps57.cd.percona.com/view/QA/job/test-ps-docker-image/58/
PS 80 : https://ps80.cd.percona.com/view/QA/job/test-ps-docker-image/186/

**Review by eleo007 — APPROVED** [review]

**Pull request merged**